src/CoinbaseSmartWallet.sol

```diff
@@ -285,21 +290,11 @@ contract CoinbaseSmartWallet is MultiOwnable, UUPSUpgradeable, Receiver, ERC1271
 
     /// @inheritdoc ERC1271
     ///
-    /// @dev Used both for classic ERC-1271 signature AND `UserOperation` validations.
-    /// @dev Reverts if the signer (based on the `ownerIndex`) is not compatible with the signature.
-    /// @dev Reverts if the signature does not correspond to an ERC-1271 signature or to the abi
-    ///      encoded version of a `WebAuthnAuth` struct.
-    /// @dev Does NOT revert if the signature verification fails to allow making a "simulation call"
-    ///      without a valid signature.
+    /// @dev Used both for ERC-1271.isValidSignature AND ERC-4337.validateUserOp signature validation.
+    /// @dev Reverts if owner at `ownerIndex` is not compatible with `signature` format.
     ///
-    /// @param signature The abi encoded `SignatureWrapper` struct.
-    function _validateSignature(bytes32 message, bytes calldata signature)
-        internal
-        view
-        virtual
-        override
-        returns (bool)
-    {
+    /// @param signature ABI encoded `SignatureWrapper`.
+    function _isValidSignature(bytes32 hash, bytes calldata signature) internal view virtual override returns (bool) {
```

Comment:
I could be convinced to revert this function name change, to avoid confusion with `ERC1271.isValidSignature`. But upon re-reviewing, I felt that `_validateSignature` is something you might call and expect a revert if it is to valid, and this name is more clear to expecting a boolean return value.

Reply:
To me renaming to `_isValidSignature` makes sense.